src/main/PileupTrack.js

```diff
@@ -165,6 +166,22 @@ function yForRow(row) {
   return row * (READ_HEIGHT + READ_SPACING);
 }
 
+// This is adapted from IGV.
+var MIN_Q = 5,  // these are Phred-scaled scores
+    MAX_Q = 20;
+function opacityForQuality(quality: number): number {
+  var alpha = 0;
+
+  if (quality < MIN_Q) {
+    alpha = 0.1;
+  } else {
+    alpha = Math.max(0.1, Math.min(1.0, 0.1 + 0.9 * (quality - MIN_Q) / (MAX_Q - MIN_Q)));
```

Comment:
correct me if I am wrong, but this can be simplified by using `d3.scale.linear`, right?
I understand what this does, but the d3 alternative might be easier to read :)